src/resources/js/tickets-attendees.js

```diff
@@ -63,6 +63,21 @@ var tribe_event_tickets_attendees = tribe_event_tickets_attendees || {};
 			return confirm( Attendees.confirmation );
 		});
 
+		$( '.attendees-admin-form' ).on( 'submit', function ( e ) {
+
+			// If not the delete action, return.
+			if ( 'delete_attendee' !== $( '#bulk-action-selector-top' ).val() ) {
+				return;
+			}
+
+			// If no attendee was selected, bail out.
+			if ( ! $( this ).serialize().includes( '&attendee' ) )  {
```

Comment:
Do we still support IE11? If we do, the `.includes()` function will not work on IE11. Instead, we'll want to use `.indexOf()`.

If we don't care about IE11, then disregard this comment ;)

Reply:
We're no longer supporting IE11 https://docs.theeventscalendar.com/wiki/view/Crossbrowser_Testing